testsIT/src/test/scala/com/stratio/crossdata/driver/DriverIT.scala

```diff
@@ -93,11 +93,11 @@ class DriverIT extends EndToEndTest {
 
   }
 
-
+  /* TODO check this tests with HDFS
   it should "be able to execute ADD JAR Command of an existent file" in {
     val file=File("/tmp/jar").createFile(false)
     val driver = Driver.getOrCreate()
-    val result = driver.addJar(s"/tmp/jar").waitForResult()
+    val result = driver.addJar(s"/home/jjlopez/Stratio/workspaceJDBC/JDBCPackage/CrossdataJDBC_v1.2.0-SNAPSHOT.tar.gz").waitForResult()
```

Comment:
It's a test with an example file created previously. It could be anything, but it is created before as an example